### ISSUE TITLE
[0.12.01] fatal error: unexpected signal during runtime execution

### ISSUE THREAD

**Issue by dsw6** [influxdata/influxdb#6442]
### Bug report

The system was running for 1+ days ingesting data every 5 seconds.    Continuous queries run every 15 and 60 minutes.  

__System info:__  
Influx 12.0.1
Ubuntu 14.04.4 LTS

[influxd.log-20160421.gz](https://github.com/influxdata/influxdb/files/230365/influxd.log-20160421.gz)

log snippet (full log attached):
-------------------------------------
```
fatal error: unexpected signal during runtime execution
[signal 0xb code=0x80 addr=0x0 pc=0x42ddba]

runtime stack:
runtime.MSpan_Sweep(0x7f53002a7470, 0x23c900002300, 0x100001501)
	/usr/local/go/src/runtime/mgc0.c:963 +0x14a

goroutine 3 [running]:
runtime.switchtoM()
	/usr/local/go/src/runtime/asm_amd64.s:198 fp=0xc20801bfa8 sp=0xc20801bfa0
runtime.gosweepone(0x0)
	/usr/local/go/src/runtime/mgc0.c:1168 +0x1e fp=0xc20801bfc0 sp=0xc20801bfa8
runtime.bgsweep()
	/usr/local/go/src/runtime/mgc0.go:85 +0x36 fp=0xc20801bfe0 sp=0xc20801bfc0
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2232 +0x1 fp=0xc20801bfe8 sp=0xc20801bfe0
created by gc
	/usr/local/go/src/runtime/mgc0.c:1386

goroutine 1 [chan receive, 3435 minutes]:
main.(*Main).Run(0xc208038340, 0xc20800a010, 0x4, 0x4, 0x0, 0x0)
	/tmp/influxdb/src/github.com/influxdata/influxdb/cmd/influxd/main.go:94 +0x765
main.main()
	/tmp/influxdb/src/github.com/influxdata/influxdb/cmd/influxd/main.go:45 +0xdc

goroutine 20 [chan receive, 3428 minutes]:
github.com/influxdata/influxdb/services/httpd.func·002()
	/tmp/influxdb/src/github.com/influxdata/influxdb/services/httpd/handler.go:298 +0x48
created by github.com/influxdata/influxdb/services/httpd.(*Handler).serveQuery
	/tmp/influxdb/src/github.com/influxdata/influxdb/services/httpd/handler.go:300 +0xd53
```

**Comment by e-dard** [comment]
@dsw6 I think that the `influxd` process may have received a `SIGSEGV` signal. If that was the case, this is the expected behaviour. Can you confirm if any signals were sent to the process?

**Comment by dsw6** [comment]
@e-dard  Sorry for the delay in responding.  I've been out of town.  

My understanding is the SIGSEGV signal is sent to a process when it makes an invalid virtual memory reference (segmentation fault).  That means the influx process triggered a segmentation fault, which caused the OS to send the signal.

**Comment by jwilder** [comment]
This might be a go 1.4.3 GC issue.  @dsw6 If this is reproducible, would you be able to test with a build compiled with go 1.6.2?

**Comment by dsw6** [comment]
The system has been running fine for multiple days since thecrash.  I do not know how to reproduce the problem.  However, I'm willing to run with a test build, if desired.

**Comment by jwilder** [comment]
Should be fixed with #6534
